src/aleph/vm/orchestrator/views/authentication.py

```diff
@@ -89,12 +91,23 @@ def check_expiry(cls, values) -> dict[str, bytes]:
     @root_validator(pre=False, skip_on_failure=True)
     def check_signature(cls, values) -> dict[str, bytes]:
         """Check that the signature is valid"""
-        signature: bytes = values["signature"]
+        signature: list = values["signature"]
         payload: bytes = values["payload"]
         content = SignedPubKeyPayload.parse_raw(payload)
-        if not verify_wallet_signature(signature, payload.hex(), content.address):
-            msg = "Invalid signature"
-            raise ValueError(msg)
+        if content.chain == "SOL":
+            from solathon.utils import verify_signature
```

Comment:
Is it the best practice to import `solathon` here ? What about trying to import it in the imports section else display a warning ? If this is a required feature, should a normal import be used instead ?

Reply:
Correct, was a oversight